bpython/sample-config

```diff
@@ -61,6 +61,7 @@
 
 # Enable autoreload feature by default (default: False).
 # default_autoreload = False
+# brackets_completion = True
```

Comment:
For everything else in this file, the commented value is the default. It could be ok to use True here, but you should add a comment above it saying that the default is False.
```suggestion

# Auto-closing brackets (default: False)
# brackets_completion = True
```

Reply:
i will change that to False. this could be misleading